packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -843,6 +843,7 @@ export const GridMixin = (superClass) =>
         level: this.__getRowLevel(row),
         expanded: this._isExpanded(row._item),
         selected: this._isSelected(row._item),
+        expandable: this._isExpandable(row._item),
```

Comment:
If this uses `itemHasChildrenPath`, should this be `hasChildren` then? Would make it easier to correlate the two and also feels like a more fitting name in general.